sceptre/cli/dump.py

```diff
@@ -114,6 +114,47 @@ def dump_template(ctx, to_file, no_placeholders, path):
             write(template, output_format, no_colour=True)
 
 
+@dump_group.command(name="vars")
+@click.argument("path")
+@click.option(
+    "--to-file", is_flag=True, help="If True, also dump the template to a local file."
+)
+@click.pass_context
+@catch_exceptions
+def dump_vars(ctx, to_file, path):
+    """
+    Dump the template vars.
+    \f
+
+    :param path: Path to execute the command on or path to stack group
+    """
+    context = SceptreContext(
+        command_path=path,
+        command_params=ctx.params,
+        project_path=ctx.obj.get("project_path"),
+        user_variables=ctx.obj.get("user_variables"),
```

Comment:
These are the user variables right here! You don't really need any of the other code to access it! It's already computed by this point!